ArcadiaParties.CQRS/Handlers/GetUserRolesHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.CQRS.Handlers
+{
+    public class GetUserRolesHandler : IRequestHandler<GetUserRolesQuery, IEnumerable<string>>
+    {
+        private readonly DataContext _context;
+
+        public GetUserRolesHandler(DataContext context)
+        {
+            _context = context;
+        }
+        public async Task<IEnumerable<string>> Handle(GetUserRolesQuery request, CancellationToken cancellationToken)
+        {
+            var user = await _context.Users.Include(u => u.UserRoles).ThenInclude(r => r.Role).FirstOrDefaultAsync(u => u.Identity == request.UserIdentity);
```

Comment:
It will look better if you divide this long line by operators:
```cs
await _context.Users
    .Include(u => u.UserRoles)
    .ThenInclude(r => r.Role)
    .FirstOrDefaultAsync(u => u.Identity == request.UserIdentity);
```

Reply:
I've formatted code